gluetool/glue.py

```diff
@@ -1488,27 +1525,62 @@ def _eval_context(self):
         return context
 
     #
-    # Module loading
+    # Module discovery and loading
     #
-    def _check_module_file(self, mfile):
+    def _register_module(self, registry, group_name, klass, filepath):
+        # type: (ModuleRegistryType, str, Type[Module], str) -> None
+        """
+        Register one discovered ``gluetool`` module.
+
+        :param dict(str, DiscoveredModule) registry: module registry to add module to.
```

Comment:
Technically, yes. I'm not sure what happens with the final HTML, how it's going to look like when rendered, but we can try.

Reply:
I decided to keep using the Sphinx format - switching to Mypy type would be helpful, but it can be done across more files than just `glue.py` and this patch - definitely worthy cleanup action, but I'd rather do it in an extra patch in the future.